### DISCUSSION TITLE
javadoc missing lots of explanations

### DISCUSSION THREAD

**Discussion by guykatz** [microstream-one/microstream#669] [Q&A] [answered]
Hi all;
in the Java Doc API many interfaces/classes are not expalined.
for example: interface PersistenceObjectManager but this is just an example.
is there work on this non documented java docs to complete it?
thanks

**Comment by hg-ms** [comment, marked as answer]
Hi,
adding more documentation on existing code currently has a very low priority although it is desirable. So please don’t expect any updates of code documentation soon.

**Reply by guykatz** [reply]
10x
